keeper/pending.go

```diff
@@ -0,0 +1,132 @@
+package keeper
+
+import (
+	"context"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+
+	"github.com/strangelove-ventures/poa"
+)
+
+// DefaultPendingValidators
+func DefaultPendingValidators() poa.PendingValidators {
+	return poa.PendingValidators{
+		Validators: []poa.Validator{},
```

Comment:
Can we use cosmos/staking/v1beta1/staking.tx for these? (instead of custom definitions)

if custom, it should be noted in an easy to copy-paste proto

Reply:
https://github.com/strangelove-ventures/poa/issues/40